monitoring/timedrift.go

```diff
@@ -136,7 +136,8 @@ func (c *timeDriftChecker) check(ctx context.Context, r health.Reporter) (err er
 	for _, node := range nodes {
 		drift, err := c.getTimeDrift(ctx, node)
 		if err != nil {
-			return trace.Wrap(err)
+			log.WithError(err).Debug("Failed to get time drift.")
```

Comment:
Should this be logged at Error level then?

Reply:
I'd like to avoid filling the logs with this error as well. In the scenario mentioned above, this error would constantly be logged while node-1's agent is down.

I think in the cases when the checker would run into this error there should be other indication that there is an issue with the cluster. If a planet-agent is down, the node should show as offline, if tcp is dropped/rejected, the node should be offline as well.

Reply:
Okay, makes sense.